phoenixRest/tests/views/agenda/__init__.py

```diff
@@ -23,6 +31,7 @@ def test_create_modify_delete_agenda(testapp):
         'description': "Test description",
         'location': "Test location",
         'time': int(datetime.datetime.now().timestamp()),
+        'duration': int(0),
```

Comment:
int(0)? Why?

Reply:
Again, I dont know why I suddenly started to use `int()`, I have not used it before. I see that it has no effect, so I removed it.